### ISSUE TITLE
Auto complete tag not showing on top (blocked by webui)

### ISSUE THREAD

**Issue by ThanapatSornsrivichai** [DominikDoom/a1111-sd-webui-tagcomplete#39]
As you can see the tag not display over the Webui and was hidden behide it.
Run on colab.
![image](https://user-images.githubusercontent.com/36567807/197728757-953ee651-5969-45a9-b06e-3cee3610f85c.png)

**Comment by DominikDoom** [comment]
I'm not able to reproduce this locally and the UI looks a bit different as well, could you tell me which colab you are using?

**Comment by ThanapatSornsrivichai** [comment]
It's this colab.
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Comment by byzod** [comment]
I run into this sometimes, the web page load slow and can't open browser console

Reloading is not working mostly, close tab and open again is needed

Besides the tagcomplete option shows rather slow than before, but I suppose it's some performance issue of webui itself, or the extension system

**Comment by DominikDoom** [comment]
> Besides the tagcomplete option shows rather slow than before, but I suppose it's some performance issue of webui itself, or the extension system (I'm not sure but the slow loading feels like only happens after tagcomplete changes to extension mode..?)

Extensions should have no influence on loading speed, they get their javascript added to the page just like the scripts in root.
Rather, it's likely because I recently updated the script to load all files async. It seems to be a bit slower than synchronous in general, I feel that too. Problem is that synchronous XMLHttpRequests are deprecated and will likely go out of support in the future (see https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Synchronous_and_Asynchronous_Requests#synchronous_request). It's not as relevant for our case where we are requesting local files instead of web links, but Chrome was showing a warning about it so I changed it.
